docs/dev-guide/integration/events/index.rst

```diff
@@ -9,15 +9,46 @@ status. This is enabled by setting ``event_notifications_enabled`` under
 ``event_notification_url`` may be set if the notification AMQP server is not
 running locally on port 5672.
 
+.. warning::
+   It is highly recommended to use a version of ``python-kombu`` that includes
+   a `fix <https://github.com/celery/kombu/issues/455>`_ for a connection leak.
+   ``python-kombu-3.0.24-6.pulp`` and newer are known to include this fix.
+
+
 Messages are published to the ``pulp.api.v2`` topic exchange. Currently only
 task status updates are published; they have a routing key of ``tasks.<task
 uuid>``. This allows a message consumer to only subscribe to updates for tasks
 they are interested in.
 
-The body of the message is a TaskStatus object in JSON form. It typically
+The body of the message is a :ref:`task_report` object in JSON form. It typically
 contains the task's ID, task status and detailed information about the task's
 progress.
 
+This example script will read all events from the exchange and print them:
+
+::
```

Comment:
This does not compile in the way that you expect. I think its missing indents. Try rewriting it and compiling the docs to check that it looks right when compiled.